### PR TITLE
Add report bug button to the settings panel

### PR DESCRIPTION
**Pull request by keianhzo**
CXB-205

### CODE REVIEW

**Comment by bluemarvin** [comment]
Looks okay. The biggest issue is the text is so small its hard to read on my Pixel 2. But that is actually true for everything in the pref pop-up

**Comment by keianhzo** [comment]
Agree, I just tried to mimic the design spec but we can always move it closer to the user if required and design agrees.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -15,12 +15,14 @@
...
     <string name="settings_telemetry">Telemetry</string>
     <string name="settings_privacy">Privacy Policy</string>
+    <string name="settings_report_issue">Report an issue\n mzl.la/fxr</string>
     <string name="settings_key_crash">settings_crash</string>
     <string name="settings_key_locale">settings_locale</string>
     <string name="settings_key_telemetry">settings_telemetry</string>
     <string name="settings_key_session_id">session_id</string>
     <string name="settings_key_previous_session_id">previous_session_id</string>
     <string name="private_policy_url">https://www.mozilla.org/en-US/privacy/firefox/</string>
+    <string name="private_report_url">https://mzl.la/fxr</string>
```
[the remark this answers is not in the document]
Comment (bluemarvin) [reply app/src/main/res/values/strings.xml]: I wish GitHub allowed you to just turn comments into issues. Would you file issues for both these comments?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -15,12 +15,14 @@
     <string name="settings_crash_reporting">Crash Reporting</string>
     <string name="settings_telemetry">Telemetry</string>
     <string name="settings_privacy">Privacy Policy</string>
+    <string name="settings_report_issue">Report an issue\n mzl.la/fxr</string>
```
[the remark this answers is not in the document]
Comment (keianhzo) [reply app/src/main/res/values/strings.xml]: Shouldn't, that depends on the Text alignment, looking good to me.

File: app/src/main/res/values/strings.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (keianhzo) [reply app/src/main/res/values/strings.xml]: Added: #150

**Pull request merged**